<launch>

    <!-- Enable custom logger configuration-->
    <env name="ROS_PYTHON_LOG_CONFIG_FILE" value="$(find mapc_rhbp_ettlinger)/launch/python_logging.conf"/>

    <!--Infrastructure Nodes-->
    <include file="$(find mapc_rhbp_ettlinger)/launch/graphhopper.launch">
        <arg name="initial_port" value="8989" />
    </include>

    <!--Planner Nodes-->
    <include file="$(find mapc_rhbp_ettlinger)/launch/planner.launch" pass_all_args="true">
    </include>


    <!-- Drones -->
        <!-- Bridges -->
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB1" />
        <arg name="only_agent_specific" value="False" />
    </include>

    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB2" />
    </include>

    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB3" />
    </include>

    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB4" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB5" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB6" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB7" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB8" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB9" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB10" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB11" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB12" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB13" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB14" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB15" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB16" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB17" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB18" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB19" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB20" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB21" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB22" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB23" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB24" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB25" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB26" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB27" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB28" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB29" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB30" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB31" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB32" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB33" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB34" />
    </include>

    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB1" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB2" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB3" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB4" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB5" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB6" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB7" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB8" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB9" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB10" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB11" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB12" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB13" />
        <arg name="bid_for_auctions" value="True" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB14" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB15" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB16" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB17" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB18" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB19" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB20" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB21" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB22" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB23" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB24" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB25" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB26" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB27" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB28" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB29" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB30" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB31" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB32" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB33" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="agentB34" />
    </include>
</launch>